src/Microsoft.TemplateEngine.Cli/Commands/InstantiateCommand.HelpUtils.cs

```diff
@@ -0,0 +1,346 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.CommandLine;
+using System.CommandLine.Help;
+using System.Text;
+
+namespace Microsoft.TemplateEngine.Cli.Commands
+{
+    internal partial class InstantiateCommand
```

Comment:
Because we got rid of the `IHelpBuilder` interface, I'm inclined to make `GetTwoColumnRow` public in-place. It has dependencies on other `HelpBuilder` state (such as per-symbol customizations) so this would be the simplest thing.